<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(236, 236, 226);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="CContextFrame" name="CContextFrame_CaptureTab_rawBuf0_FFT">
        <layout class="QVBoxLayout" name="verticalLayout_Capture_FFT">
         <item>
          <widget class="CContextFrame" name="CContextFrame_Stamps">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_stamps_0">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>557</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="CLabel" name="CLabel_acqStamp_Capture">
              <property name="text">
               <string>&lt;b&gt;acqStamp:&lt;/b&gt; Null  </string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="CLabel" name="CLabel_cycleName_Capture">
              <property name="text">
               <string>&lt;b&gt;cycleName:&lt;/b&gt; Null</string>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_stamps_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>557</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBox_one">
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator {
    width: 18px; height: 18px;
}

QCheckBox::indicator:checked {
    image: url(icons/checkbox_true_yellow_2.png);
}

QCheckBox::indicator:unchecked {
    image: url(icons/checkbox_false_2.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(icons/checkbox_true_pressed_2.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(icons/checkbox_false_pressed_2.png);
}

QCheckBox::indicator:checked:disabled {
    image: url(icons/checkbox_true_disabled_2.png);
}

QCheckBox::indicator:unchecked:disabled {
    image: url(icons/checkbox_false_disabled_2.png);
}</string>
             </property>
             <property name="text">
              <string>Peaks</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_hover">
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator {
    width: 18px; height: 18px;
}

QCheckBox::indicator:checked {
    image: url(icons/checkbox_true_green_2.png);
}

QCheckBox::indicator:unchecked {
    image: url(icons/checkbox_false_2.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(icons/checkbox_true_pressed_2.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(icons/checkbox_false_pressed_2.png);
}

QCheckBox::indicator:checked:disabled {
    image: url(icons/checkbox_true_disabled_2.png);
}

QCheckBox::indicator:unchecked:disabled {
    image: url(icons/checkbox_false_disabled_2.png);
}</string>
             </property>
             <property name="text">
              <string>Datapoints (Hover)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sync_main">
             <property name="styleSheet">
              <string notr="true">QCheckBox::indicator {
    width: 18px; height: 18px;
}

QCheckBox::indicator:checked {
    image: url(icons/checkbox_true_white_2.png);
}

QCheckBox::indicator:unchecked {
    image: url(icons/checkbox_false_2.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(icons/checkbox_true_pressed_2.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(icons/checkbox_false_pressed_2.png);
}

QCheckBox::indicator:checked:disabled {
    image: url(icons/checkbox_true_disabled_2.png);
}

QCheckBox::indicator:unchecked:disabled {
    image: url(icons/checkbox_false_disabled_2.png);
}</string>
             </property>
             <property name="text">
              <string>Sync (UCAP)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_CValueAggregators">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>CContextFrame</class>
   <extends>QFrame</extends>
   <header>comrad.widgets.containers</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>PyDMLabel</extends>
   <header>comrad.widgets.indicators</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
